pkg/doc/signature/jsonld/processor.go

```diff
@@ -0,0 +1,163 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package jsonld
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/piprate/json-gold/ld"
+
+	"github.com/hyperledger/aries-framework-go/pkg/common/log"
+)
+
+const (
+	format             = "application/n-quads"
+	algorithm          = "URDNA2015"
```

Comment:
should  any of these be options?

Reply:
will fix this, but right now json-gold only support this algorithm. Not sure if this algorithm is something to do with suite.

[Universal RDF Dataset Normalization Algorithm 2015 or URDNA2015.](http://json-ld.github.io/normalization/spec/)

Reply:
if your below logic is tied to URDNA - eg retryForInvalidRDF then i guess it couldn’t be a parameter.

Reply:
done